draft-ietf-rats-corim.md

```diff
@@ -1259,6 +1259,22 @@ for appraisal until a CoBOM which activates that tag has been received and
 successfully processed. All the tags listed in the CoBOM must be activated in
 the same transaction, i.e., either all or none.
 
+The number of CoBOMs required in a given supply chain eco-system is dependent on
+Verifier Owner's appraisal policy, which is often driven by the complexity and nature of
+use case.
+
+In a certain use case CoBOM's may not be required at all. For example, where only limited CoRIMs
+are produced by a single authority. In such cases any tags in a CoRIM received by a Verifier
+are activated immediately and treated valid for appraisal.
```

Comment:
```suggestion
Some Verifier Owners will have the policy that CoBOM's are not required.
A Verifier Owner policy which rejects CoRIMs which do not have a signature chain from an
acceptable authority may not require additional authorization via CoBOM. This Verify Owner
would set the Verifier policy to activate all tags immediately, skipping CoBOM checks.
```

Reply:
Re-worded and tidied up as per Andrew's suggestion!